src/main/java/rx/internal/util/ScalarSynchronousObservable.java

```diff
@@ -49,5 +53,81 @@ public void call(Subscriber<? super T> s) {
     public T get() {
         return t;
     }
+    
+//    @Override
```

Comment:
I've experimented with disabling the ```final``` in ```Observable``` so I could just override the method, but that made the call site megamorphic and lost performance I gained with specialization. The current code in ```Observable``` is just bimorphic. I will remove these lines.